### PR TITLE
Addressed comments from PR 154

### PR DESCRIPTION
**Pull request by AndrewDorn**
## Summary of Changes

Addresses comments from PR 154
https://github.com/nasa/cumulus-orca/pull/154/

## Changes

* Detailed list or prose of changes
* ...

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests (unit, integration, ad-hoc, or otherwise) that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the necessary documentation (remove those that do not apply)
    - [x] CHANGELOG.md
    - [x] Testing documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
    - [x] Unit tests
    - [ ] Integration tests
    - [ ] Manual tests (outlined above)
- [x] New and existing unit tests pass locally with my changes
    - [x] Automated tests passing (if not fork)
    - [ ] Manual testing/integration testing passes (if forked)
- [x] Any dependent changes have been merged and published in downstream modules
- [x] My code has passed security scanning
    - [x] Snyk
    - [x] git-secrets

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -34,6 +33,7 @@ module "orca" {
 
   ## OPTIONAL
   database_port                                        = var.database_port
+  default_multipart_chunksize_mb = var.default_multipart_chunksize_mb
```
Comment (rizbi-hassan) [inline main.tf:36]: maybe add some spacing to the formatting

File: tasks/db_deploy/test/manual_tests/sql/orca_schema_v3/remove.sql [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
 DELETE FROM orca.schema_versions
     WHERE version_id=3
-GO
-UPDATE orca.schema_versions
-    SET is_latest=true
-    WHERE version_id=2
-GO
-COMMIT;
+-- Upsert the old version
+INSERT INTO schema_versions
```
Comment (rizbi-hassan) [inline tasks/db_deploy/test/manual_tests/sql/orca_schema_v3/remove.sql:9]: should this be `INSERT INTO orca.schema_versions`?

File: variables.tf [hunk 3]
```diff
@@ -92,6 +85,13 @@ variable "database_port" {
 }
 
 
+variable "default_multipart_chunksize_mb" {
+  type        = number
+  description = "The default maximum size of chunks to use when copying. Can be overridden by collection config."
+  default     = "250"
```
Comment (rizbi-hassan) [inline variables.tf:91]: "" is not needed in default since it's a number

**Review by rizbi-hassan — CHANGES_REQUESTED** [review]
few minor things I found

File: main.tf (hunk 1 above)
Comment (AndrewDorn) [reply main.tf:36]: Fixed

File: tasks/db_deploy/test/manual_tests/sql/orca_schema_v3/remove.sql (hunk 2 above)
Comment (AndrewDorn) [reply tasks/db_deploy/test/manual_tests/sql/orca_schema_v3/remove.sql:9]: Fixed

File: variables.tf (hunk 3 above)
Comment (AndrewDorn) [reply variables.tf:91]: Fixed

**Review by bhazuka — APPROVED** [review]

**Review by rizbi-hassan — APPROVED** [review]

**Pull request merged**